Makefile

```diff
@@ -10,8 +10,8 @@ all:
 	@echo ""
 	@echo "To install it try \"make install\" instead."
 	@echo
-	@echo "To run pass $(PROG) one needs to have some tools installed on the system:"
-	@echo "     Tomb and password store"
+	@echo "To run pass-$(PROG) one needs to have some tools installed on the system:"
```

Comment:
`pass-tomb` is the name of the project. However as a pass extension the command is `pass tomb`. Therefore the change is not needed